src/components/dev-hub/event-list.js

```diff
@@ -13,6 +13,7 @@ const EventsPreview = styled('div')`
     display: flex;
     flex-direction: row;
     justify-content: space-between;
+    min-height: ${size.xxlarge};
```

Comment:
This only solves it for the events preview on desktop. Mobile we could also do a min-height but since they are stacked it seemed a bit more natural?

Reply:
Yea that makes sense and is a good decision 👍